archive-data/archive.py

```diff
@@ -0,0 +1,204 @@
+"""This file is responsible for moving old data from
+the short-term storage to the long-term storage."""
+
+from functools import reduce
+import logging
+from os import environ as ENV
+
+from dotenv import load_dotenv
+from psycopg2 import connect, Error
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+import pandas as pd
+
+from archive_queries import (S_DELAYS, S_DELAYS_OVER_5_MIN, S_AVG_DELAY, S_TOTAL_ARRIVALS, S_TOTAL_CANCELLATIONS,
+                             O_DELAYS, O_DELAYS_OVER_5_MIN, O_AVG_DELAY, O_TOTAL_ARRIVALS, O_TOTAL_CANCELLATIONS,
+                             INSERT_STATION_PERFORMANCE, INSERT_OPERATOR_PERFORMANCE,
+                             DELETE_OLD_ARRIVAL_DATA, DELETE_OLD_CANCELLATION_DATA)
+
+
+class DataFetchError(Exception):
+    """Exception raised when there is an error fetching data."""
+    pass
+
+
+def setup_logging() -> None:
+    """Sets up the logging configuration."""
+
+    logging.basicConfig(level=logging.INFO,
```

Comment:
Is the timestamp for logs in BST or UCT?

Reply:
👀 UCT?